docs/hf_hub.rst

```diff
@@ -111,3 +111,42 @@ returns the model output for a given input. It can be used as::
 
 In the above code snippet, ``res`` will be a :class:`numpy.ndarray` containing
 the model's output.
+
+.. _hf_hub_gradio:
+
+Easily build user interfaces to your scikit-learn models
+--------------------------------------------------------
+`gradio <https://gradio.app/>`__ is a python library that lets you create interfaces on your model.
+It has a class called :class:`Interface` that lets you create application
+interfaces to your machine learning models.
+``gradio`` is integrated with skops, so you can load an interface with only one
+line of code. You need to initialize an interface, call load method with
```

Comment:
The sentence structure seems to be a bit weird to me:

> You need to initialize an interface, call load method with your repository identifier (...) prepended with "huggingface/" will load an interface for your model.

Not quite sure what it should actually be. Maybe split it into 2 sentences?